src/status_im2/contexts/wallet/events.cljs

```diff
@@ -269,7 +269,7 @@
                               :max-cache-age-seconds max-cache-age-seconds}
          request-params      [request-id
                               [(chain/chain-id db)]
-                              (map :address (:profile/wallet-accounts db))
+                              (keys (get-in db [:wallet :accounts]))
```

Comment:
@mmilad75, here is a fix that enables collectibles retrieving.
Old wallet initialization was failed and `:profile/wallet-accounts` key is not longer populated, so now we need to get account information in a new way

Reply:
Great! Thanks man.

Reply:
Apologies 🙏 for not spotting this yesterday when we removed it.